templates/layouts/main.html

```diff
@@ -78,7 +78,9 @@
       <p class="address__title">Federal Election Commission</p>
       <p>999 E Street, NW, Washington, DC 20463</p>
       <p>(800) 424-9530</p>
-      <p><a href="mailto:{{ contact_email }}">{{ contact_email }}</a></p>
+      <p class="address__subtitle">betaFEC feedback</p> 
+      <p><a href="mailto:{{ contact_email }}">{{ contact_email }}</a><img src="img/i-email--inverse.svg" alt="Icon of email being sent"></p>
+      <p><a href="https://github.com/18F/fec">github.com/18F/FEC</a><img src="img/i-github--inverse.svg" alt="Icon of GitHub logo"></p>
```

Comment:
Pinging @emileighoutlaw for review on the GitHub footer link text, as it was her call to use the same format as 18F's footer link in https://github.com/18F/openFEC-web-app/issues/866#issuecomment-151214687 . I definitely see the case for making both the footer and contact page match, whichever format we go with.